rpctest/blockgen.go

```diff
@@ -125,8 +125,8 @@ func createBlock(prevBlock *btcutil.Block, inclusionTxs []*btcutil.Tx,
 	// it's the genesis block in which case use the current time.
 	var ts time.Time
 	switch {
-	case blockTime != nil:
-		ts = *blockTime
+	case !blockTime.IsZero():
+		ts = blockTime
 	case blockHeight == 0:
```

Comment:
Shouldn't this be impossible?  You always have to build off at least the genesis block which is at height and given `blockHeight` is `prevBlock.Height() + 1`, that means it must always be at least 1.